docs/modules/ROOT/pages/how-to-guides/managing-applications/proc-managing_applications.adoc

```diff
@@ -0,0 +1,93 @@
+= Secure development environment
```

Comment:
Unsure about why this title was used 🤔

Reply:
And for whatever title you end up using, you'll want to make sure to add the new page to the nav bar as well :+1:

Reply:
Hi @Chr1st1anSears this topic won't be added to the navigation bar since this is internal only content.